src/hooks.ts

```diff
@@ -0,0 +1,46 @@
+import type { GetSession, Handle } from '@sveltejs/kit';
+import * as cookie from 'cookie';
+import sessionDB from '$lib/Utils/sessionDB';
+
+export const handle: Handle = async ({ request, render }) => {
+	const cookies = cookie.parse(request.headers.cookie || '');
+
+	if (!cookies.session_id || cookies.session_id === 'deleted') request.locals.authenticated = false;
+	else {
+		request.locals.authenticated = true;
+		request.locals.session_id = cookies.session_id;
+	}
+
+	console.log(
```

Comment:
Je hebt wel op meer plekken een log staan. Ik zelf vind het niet zo fijn. Het is alleen met testen handig, lijkt mij?

Reply:
ik wil deze er nog even in laten staat, ik snap namelijk redelijk goed wat er gebeurd met die sessies en authenticatie binnen svelte, maar nog niet helemaal. (en ergens een bugje van svelte )